Expensify/App
# send button bug resolved

### Details
There was a bug in the cross-platform React Native application on all platforms (web, iOS, Android, desktop) where the send button is the same color with and without text input.

Without text input, we need this to be grey. When there is text in text input, this needs to turn to green.

Reproduction steps:
1. Open the app and login
2. Observe the 'Send' button without any text input and notice it is green
3. Add text and notice that it does not change color
4. Remove all of the text and observe that it remains green

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/983

### Tests
Test 1
1. Open the app and log in
2. Observe the 'Send' button without any text input and notice it is gray
3. Add text and notice that now it does change color (Previously it was not)
4. Remove all of the text and observe that it again becomes gray (previously it was green)

Test 2
1. Open the app and log in
2. Observe the 'Send' button without any text input and notice it is gray
3. Add text and notice that now it does change color (Previously it was not)
4. Before clicking on the Send button, open up another Chat Thread and type some text into that Thread.
5. Again comes back to the original Thread, Sees that We have Send Button in Green.



### Screenshots
Attaching a Video For Showing the Fix
https://user-images.githubusercontent.com/35005181/103121447-2be4e700-46a2-11eb-81fe-96731d1e6812.mov
